src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SimpleConfigModel.cs

```diff
@@ -477,170 +632,89 @@ public void Evaluate(IParameterSet parameters, IVariableCollection rootVariableC
                 {
-                    SourceModifier modifier = new SourceModifier
-                    {
-                        Condition = entry.ToString(nameof(modifier.Condition)),
-                        CopyOnly = entry.Get<JToken>(nameof(modifier.CopyOnly)),
-                        Exclude = entry.Get<JToken>(nameof(modifier.Exclude)),
-                        Include = entry.Get<JToken>(nameof(modifier.Include)),
-                        Rename = entry.Get<JObject>(nameof(modifier.Rename))
-                    };
-                    modifiers.Add(modifier);
-                }
-
-                src.Source = item.ToString(nameof(src.Source));
-                src.Target = item.ToString(nameof(src.Target));
-            }
-
-            IBaselineInfo baseline = null;
-            config.BaselineInfo = BaselineInfoFromJObject(source.PropertiesOf("baselines"));
-
-            if (configModifiers != null && !string.IsNullOrEmpty(configModifiers.BaselineName))
-            {
-                config.BaselineInfo.TryGetValue(configModifiers.BaselineName, out baseline);
-            }
-
-            Dictionary<string, ISymbolModel> symbols = new Dictionary<string, ISymbolModel>(StringComparer.Ordinal);
-            // create a name symbol. If one is explicitly defined in the template, it'll override this.
-            symbols.Add(NameSymbolName, SetupDefaultNameSymbol(config.SourceName));
-
-            // tags are being deprecated from template configuration, but we still read them for backwards compatibility.
-            // They're turned into symbols here, which eventually become tags.
-            config._tags = source.ToStringDictionary(StringComparer.OrdinalIgnoreCase, "tags");
-            IReadOnlyDictionary<string, ISymbolModel> symbolsFromTags = ConvertDeprecatedTagsToParameterSymbols(config._tags);
-
-            foreach (KeyValuePair<string, ISymbolModel> tagSymbol in symbolsFromTags)
-            {
-                if (!symbols.ContainsKey(tagSymbol.Key))
-                {
-                    symbols.Add(tagSymbol.Key, tagSymbol.Value);
+                    postAction.Localize(postActionLocModel, _logger);
                 }
             }
+        }
 
-            config.Symbols = symbols;
-            foreach (JProperty prop in source.PropertiesOf(nameof(config.Symbols)))
+        /// <summary>
+        /// Verifies that the given localization model was correctly constructed
+        /// to localize this SimpleConfigModel.
+        /// </summary>
+        /// <param name="locModel">The localization model to be verified.</param>
+        /// <returns>True if the verification succeeds. False otherwise.
+        /// Check logs for details in case of a failed verification.</returns>
+        internal bool VerifyLocalizationModel(ILocalizationModel locModel, out IEnumerable<string> localizedErrorMessages)
```

Comment:
(i don't think we should address it now, but we need to create issue on it)

there are more validations that can be added:
- similar checks for symbols
- warning level validations when name, description or author is not localized (if they present in base template); not localized symbols and post actions (to be included later to `--validate` option

Reply:
Created #3254